datafusion/common/src/stats.rs

```diff
@@ -433,6 +480,79 @@ fn check_num_rows(value: Option<usize>, is_exact: bool) -> Precision<usize> {
     }
 }
 
+pub fn add_row_stats(
```

Comment:
These methods had to remain `pub` for now as they are also used in the listing table statistics summary code. I will look into that and see if it makes sense to do as a individual PR